package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "setup-mabl-cli",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
We've generally been doing minor releases each time, to keep the tags simple, so this would be `v1.1` when we tag.